plugins/frontend/backstage-plugin-argo-cd/src/components/ArgoCDDetailsCard.tsx

```diff
@@ -73,17 +70,35 @@ type OverviewComponentProps = {
   retry: () => void;
 };
 
-const OverviewComponent = ({ data, extraColumns, retry }: OverviewComponentProps) => {
+const OverviewComponent = ({
+  data,
+  extraColumns,
+  retry,
+}: OverviewComponentProps) => {
   const configApi = useApi(configApiRef);
-  const baseUrl = configApi.getOptionalString('argocd.baseUrl')
+  const baseUrl = configApi.getOptionalString('argocd.baseUrl');
+  const argoScan = configApi.getOptionalConfigArray('argocd.appLocatorMethods') ? true : false;
```

Comment:
This is failing type check it looks like. Unused

Reply:
addressed